### ISSUE TITLE
[EPIC] v1.5 Site Showcase - cleanup

### ISSUE THREAD

**Issue by shannonbux** [gatsbyjs/gatsby#7488]
### Who will own this?

**Area of Responsibility:**

_Select the Area of Responsibility *most impacted* by this Epic_

- [ ] Community
- [ ] Learning
- [x] Ecosystem
- [ ] Gatsby Core
- [ ] Enterprise
- [ ] Design
- [ ] Marketing
- [ ] Company/Admin
- [ ] Sales

     |                 |       |
     | --------------- | ----- |
     | AoR owner       | @amberleyromo |
     | Domain owner   | @name |
     | Project manager | @name |
     | Tech lead       | @name |
     | Contributors    | @name |

### Summary

We need to clean up a few bugs happening with the site showcase so the user experience is smoother.

### How will this impact Gatsby?

**Domains**

.org website

**Components**

_List the impacted Components here_

### Goals

1. Reduce the number of bugs with the site showcase

### How will we know this epic is a success?

_What changes must we see, or what must be created for us to know the project was a success. How will we know when the project is done? How will we measure success?_

**User Can Statement**

- User can find the tags they are looking for
- User can view beautiful screenshots, no matter what language is on the screenshot
- User can experience a fast website, which probably requires pagination in the site showcase

**Metrics to Measure Success**

- Decrease in number of bugs

## MVP
Here's what we already finished in Issue #4392:

### What are the risks to the epic?

The risks of organizing this partially in the [EPIC] v2, [EPIC] v2 UX & docs, and this epic, could lead to confusion and mis-prioritization and even duplicate work.

### What questions do we still need to answer, or what resources do we need?

Need to assign a tech lead and contributors.

### How will we complete the epic?

* Need to assign a tech lead and contributors
* Need to decide whether site showcase bugs and requests go here or in [EPIC] v2 or [EPIC] v2 UX & docs
* Need to place this in the roadmap in the correct order
* Have a kickoff meeting when we start working on this Epic

### How else could we accomplish the same goal?

Nope!

**Comment by fk** [comment]
I think this is related to a different filtering logic (than the one we went for and is live on https://next.gatsbyjs.org/showcase/):
Given two categories a and b, and 10 total sites of which

* 2 sites are in category a _and_ b
* 4 sites are just in category a
* 4 sites are just in category b
* = a total of 5 sites in category a and b each

the sidebar filters initially would look like this

* ( ) category a 5
* ( ) category b 5

and all 10 sites being displayed.
then selecting "category a" would

* result in the 5 sites in category a being shown
* the sidebar filters to update like this:
  * (x) category a 5
  * ( ) category b **2**
  …with the count for all unselected category filters being updated to reflect the intersection of the selected category and the unselected category -- selecting "category b" now would result in the site list updating to just show sites in both category a and b (and raising the question "what should the count for each category reflect now? :-D)…

Bottom line: We went for "unity" instead of "intersection", so we can safely forget about

> when a filter is selected any category that is not related to any sites left by the filter should be hidden, as selecting them would have no effect on the state of the page

**Comment by missmatsuko** [comment]
Should this issue be re-opened? I wanted to report an issue with the Showcase screenshots but found #5929 which led me to this epic.

**Comment by shannonbux** [comment]
All the issues that are closed in this Epic could be reopened too!

**Comment by sidharthachatterjee** [comment]
Let's go ahead and close this for now since there aren't any current issues with the Site Showcase. The issue regarding screenshots was most recently fixed in https://github.com/gatsbyjs/gatsby/pull/20427

Thanks everyone!

**Comment by missmatsuko** [comment]
@sidharthachatterjee That's not true. Non-roman characters still aren't rendering (#5929) and I don't see how the solution in #20427 (expanding Lambda container's space) could possibly fix that.

**Comment by LekoArts** [comment]
The other issue is describing that so that epic here doesn’t have to stay open :)

**Comment by missmatsuko** [comment]
What's actually a part of this epic? At This issue was previously reopened because #5929 was outstanding so I thought it's part of this epic and the epic should remain open.

**Comment by sidharthachatterjee** [comment]
@missmatsuko You're absolutely right in that the open issue is part of this epic but 

- We at @gatsbyjs/core don't really use epics anymore
- Since that's the only issue open, I don't see the point of _also_ keeping this open 

Hope that makes sense.
